Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3657,11 +3657,26 @@ script:
       predefined:
       - 'true'
       - 'false'
+    - auto: PREDEFINED
+      description: Whether to use polling.
+      defaultValue: 'false'
+      name: polling
+      predefined:
+      - 'true'
+      - 'false'
+    - description: Job ID to use in polling commands.
+      name: job_id
+      deprecated: true
```

Comment:
I had to make it deprecated as I don't want to the user thinking that they had to provide it. It seems the only way to hide an input argument.

Reply:
I'll do the pack relaese notes bump shortly..

Reply:
As for the parts for .get, I only added code / copied code from existing calls. I will modify to ensure .get is used in these new calls, but I'm reluctant to change that code across other functions.

Reply:
i don't think you need to change across other functions, only those who you change because when we use ```result['response']``` it is more prone to errors than using ```result.get('response')``` 
It is not super important, i know in most of the codes its written the way you describe. That is just my recommendation, hence its not a must.